bin/compile

```diff
@@ -403,6 +412,8 @@ trap clearGitCredHelper INT TERM EXIT
 
 determineTool
 
+mcount "version.$ver"
+
 ver=$(expandVer $ver)
```

Comment:
iiuc, this may modify the `ver` variable. What if we added the `mcount` inside the `reportVer` function instead?

Reply:
We should record the initial version and the expanded version IMO.

Reply:
@jmorrell @freeformz expandVer modifies the `ver` variable with go${v} . I was thinking that as long as we capture the version we should be fine and hence I placed it there. But lets chat about this.

Reply:
The issue is that you'll capture both: `1.12` and `go1.12` and count them as separate things when you place it here.

That info is also useful, but it's probably not what you want when capturing "which go version was installed"